server/helpers/express-utils.ts

```diff
@@ -28,6 +29,19 @@ function buildNSFWFilter (res?: express.Response, paramNSFW?: string) {
   return null
 }
 
+function isPrivacyForFederation(privacy: VideoPrivacy) {
```

Comment:
Why did you move these functions in this file? `VideoModel` or `helpers/videos.ts` seems more appropriate